packages/backend/src/tools/uif/KnexMiddleware.ts

```diff
@@ -0,0 +1,24 @@
+import { Knex } from 'knex'
+import { DbTransaction } from './multi/types'
+
+export class KnexTx implements DbTransaction {
```

Comment:
Maybe let's use `KnexTrx` and in other parts of the code also call it `trx` because the naming is not consistent